### DISCUSSION TITLE
Annotations covered by hotspots

### DISCUSSION THREAD

**Discussion by designbyadrian** [google/model-viewer#3440] [Q&A]
### Description

Annotations are easily covered by other hotspots. There doesn't seem to be a clear way of using z-indexes to have them appear on a layer in front of _all_ hotspots?

#### Live Demo

<img width="470" alt="image" src="https://user-images.githubusercontent.com/2064558/154975430-34c8e79d-47d0-4c63-b823-a684166eea4c.png">

https://glitch.com/edit/#!/snow-amber-floor

### Version
- model-viewer: v1.10.1

### Browser Affected

- [ x ] Chrome: 98.0.4758.102
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ x ] Safari: 15.3

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ x ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
This feature is based on the Three.js CSS2DRenderer, which layers them according to their distance from the camera. Think of them as floating billboards, pointed at the camera but anchored to the hotspot position. Since their ordering is physical, I'd recommend cleaning this up by changing the relative positioning of your child divs and perhaps hiding/showing things dynamically.
